src/main/java/org/spongepowered/common/mixin/core/item/inventory/MixinContainerRepair.java

```diff
@@ -63,4 +85,25 @@ public SlotProvider slotProvider(Fabric fabric, InventoryAdapter adapter) {
                 .add(36);
         return builder.build();
     }
+
+    @Inject(method = "updateRepairOutput",
+            locals = LocalCapture.CAPTURE_FAILSOFT,
+            cancellable = true,
+            at = @At(value = "INVOKE_ASSIGN", target = "Lnet/minecraft/item/ItemEnchantedBook;getEnchantments(Lnet/minecraft/item/ItemStack;)Lnet/minecraft/nbt/NBTTagList;"))
```

Comment:
I'm confused as to where this is being injected to. It sounds like it's injecting right after `getEnchantments(ItemStack)` is called, but before the `NBTTagList.isEmpty()` is called. I get that the map is not being modified here, but what's the point of the map if it's not being used? Couldn't this cancellable injection occur after the isEmpty and value assignment is made?